cfme/tests/infrastructure/test_timelines.py

```diff
@@ -39,6 +43,25 @@ def mark_vm_as_appliance(new_vm, appliance):
     relations_view.form.save_button.click()
 
 
+@pytest.fixture(scope='function')
+def control_policy(appliance, new_vm):
```

Comment:
:+1:

Reply:
thats good?

Reply:
yep, I happy to see that you are going to cover policy timelines as well.

Reply:
https://github.com/ManageIQ/integration_tests/pull/7073 for Cloud intel as well ;)